package.json

```diff
@@ -2,7 +2,7 @@
   "name": "nativescript-ringer",
   "version": "1.0.0",
   "description": "A plugin for controlling the ringer volume and mode in NativeScript Android apps.",
-  "main": "index.js",
+  "main": "index.android.js",
```

Comment:
The use of `index.js` as the main value was intentional. By using `index.js`, this used to lead NativeScript to check for either `index.android.js` or `index.ios.js` and include the appropriate one:
https://v7.docs.nativescript.org/angular/code-sharing/code-splitting#ios-android-web-files

I think this is still supported, but can't seem to find it in their updated docs. That was the latest version I saw it in.

Reply:
I originally got errors when nativescript tried to load the module and this fixed them. It didn't seem to affect the actual function of the module, however it didn't seem happy with it just being index.js.